OCKSample/Main/Care/CareViewController.swift

```diff
@@ -41,6 +41,25 @@ class CareViewController: OCKDailyPageViewController {
 
     private var isSyncing = false
     private var isLoading = false
+    var events: CareStoreFetchedResults<OCKAnyEvent, OCKEventQuery>? {
+        didSet {
+            self.reloadView()
+        }
+    }
+
+    /// Create an instance of the view controller. Will hook up the calendar to the tasks collection,
+    /// and query and display the tasks.
+    ///
+    /// - Parameter store: The store from which to query the tasks.
+    /// - Parameter computeProgress: Used to compute the combined progress for a series of CareKit events.
+    init(store: OCKAnyStoreProtocol,
+         events: CareStoreFetchedResults<OCKAnyEvent, OCKEventQuery>? = nil,
+         computeProgress: @escaping (OCKAnyEvent) -> CareTaskProgress = { event in
+        event.computeProgress(by: .checkingOutcomeExists)
+    }) {
+        super.init(store: store, computeProgress: computeProgress)
+        // self.events = events
```

Comment:
This is the other line commented out due to app crash, uncomment both lines to re-enable Steps card